src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -422,9 +422,28 @@
     },
     "http": {
       "displayName": "HTTP",
-      "description": "Register native handlers for http and setup Refit support",
+      "description": "Register native handlers for http.",
       "type": "parameter",
-      "datatype": "bool"
+      "datatype": "choice",
+      "defaultValue": "Default",
```

Comment:
It probably shouldn't have a default value here. This would mean `Default` would be the default value for blank and recommended.

Instead we want to change the `presetHttpDefault` to something similar to: https://github.com/unoplatform/uno.templates/blob/2bce19094a415d46ca8875e602824d5c9dc7e8ee/src/Uno.Templates/content/unoapp/.template.config/template.json#L1055-L1072

That would also mean introducing a new option `None` option for Http, since with this setup it wouldn't be possible to not choose Http.